platform/android/export/export_plugin.cpp

```diff
@@ -1558,11 +1558,7 @@ String EditorExportPlatformAndroid::load_splash_refs(Ref<Image> &splash_image, R
 	}
 
 	// Setup the splash bg color
-	bool bg_color_valid;
-	Color bg_color = ProjectSettings::get_singleton()->get("application/boot_splash/bg_color", &bg_color_valid);
-	if (!bg_color_valid) {
-		bg_color = boot_splash_bg_color;
```

Comment:
I believe you can remove the `splash.gen.h` include which was used for `boot_splash_bg_color`.

Reply:
seems the actual png is used, so I can't remove it